interactions/api/models/channel.py

```diff
@@ -1201,10 +1228,60 @@ async def join(self) -> None:
 @define()
 class Thread(Channel):
     """An object representing a thread.
-
     .. note::
         This is a derivation of the base Channel, since a
         thread can be its own event.
+        Also note that forums are also dispatched as Thread
+        objects.
+    """
+
+    # __slots__ = (
+    #     "_json",
+    #     "id",
+    #     "type",
+    #     "guild_id",
+    #     "position",
+    #     "permission_overwrites",
+    #     "name",
+    #     "topic",
+    #     "nsfw",
+    #     "last_message_id",
+    #     "bitrate",
+    #     "user_limit",
+    #     "rate_limit_per_user",
+    #     "recipients",
+    #     "icon",
+    #     "owner_id",
+    #     "application_id",
+    #     "parent_id",
+    #     "last_pin_timestamp",
+    #     "rtc_region",
+    #     "video_quality_mode",
+    #     "message_count",
+    #     "member_count",
+    #     "thread_metadata",
+    #     "member",
+    #     "default_auto_archive_duration",
+    #     "permissions",
+    #     "_client",
+    #     # TODO: Document these when Discord officially documents them.
+    #     "banner",
+    #     "guild_hashes",
+    #     "flags",  # This seems to be forum unique.
+    #     "newly_created",
+    #     "template",
+    #     "available_tags",
+    # )
+
+    ...
+
+
+@define()
+class Forum(Thread):
+    """An object representing a Forum.
+    .. note::
+        While this isn't directly dispatched for GW based events,
+        this object exists moreso for future helper methods.
```

Comment:
Wouldn't it make more sense to keep the methods on the channel object like we did for threads?